ansible/config_sonic_basedon_testbed.yml

```diff
@@ -527,6 +527,33 @@
           timeout: 600
         changed_when: false
 
+      - name: Sync DUT system time with NTP server
+        block:
+          - name: Wait for ntp.service restart after reload minigraph
+            shell: systemctl is-active ntp.service
```

Comment:
Ansible has a `service` module which is capable of getting/setting service status. Using `systemctl` command is fine anyway.

Reply:
Thanks @wangxin, I updated it in my latest commit. `service` module really makes code clearer!